### DISCUSSION TITLE
Positioning Parts

### DISCUSSION THREAD

**Discussion by cemalgnlts** [CodinGame/monaco-vscode-api#292] [Q&A] [answered]
Hi,
I want to add the sidebar right-aligned. Of course, I already added it to the right side with CSS, but when the piece is added, it is added by calculating that it will be on the left side.

| Visually on the right: | CSS properties have been added to be based on the left side: |
|-------|---------|
|![Screenshot 2023-12-08 11 58 03](https://github.com/CodinGame/monaco-vscode-api/assets/45357531/9e508352-3c4e-46c3-85c9-a4f33f8eeb36)|![Screenshot 2023-12-08 11 58 47](https://github.com/CodinGame/monaco-vscode-api/assets/45357531/fc2a36f9-f419-42a5-b8bd-b881e67733cf)|

I tried to align it to the right like this:
```diff
{
  "workbench.colorTheme": "Default Dark Modern",
+ "workbench.sideBar.location": "right",
  "editor.fontSize": 14,
  "editor.tabSize": 2,
  "editor.scrollBeyondLastLine": false,
  "editor.minimap.enabled": false
}
```

There was no change.

**Comment by cemalgnlts** [comment]
I found this on the to-do list:

> - [ ] Ability to respect sidebar position.

https://github.com/CodinGame/monaco-vscode-api/discussions/99

**Reply by CGNonofr** [reply, marked as answer]
It will be implemented with https://github.com/CodinGame/monaco-vscode-api/pull/294
